java/cdk/src/main/java/sleeper/cdk/stack/TransactionLogSnapshotStack.java

```diff
@@ -128,7 +142,73 @@ public TransactionLogSnapshotCreationStack(
         coreStacks.grantInvokeScheduled(snapshotCreationTrigger, queue);
         coreStacks.grantReadTablesStatus(snapshotCreationLambda);
         transactionLogStateStoreStack.grantCreateSnapshots(snapshotCreationLambda);
+    }
 
-        Utils.addStackTagIfSet(this, instanceProperties);
+    private void createSnapshotDeletionLambda(InstanceProperties instanceProperties, LambdaCode statestoreJar, CoreStacks coreStacks,
+            TransactionLogStateStoreStack transactionLogStateStoreStack, Topic topic, List<IMetric> errorMetrics) {
+        String instanceId = Utils.cleanInstanceId(instanceProperties);
+        String triggerFunctionName = String.join("-", "sleeper", instanceId, "tl-snapshot-deletion-trigger");
+        String deletionFunctionName = String.join("-", "sleeper", instanceId, "tl-snapshot-deletion");
```

Comment:
Please can we make the lambda names consistent between creation and deletion? Maybe something like state-snapshot-creation, state-snapshot-creation-trigger, state-snapshot-deletion, state-snapshot-deletion-trigger?